src/governance.ts

```diff
@@ -273,6 +275,57 @@ export class GovernanceCanister {
     });
   };
 
+  /**
+   * Splits a neuron creating a new one
+   *
+   * @returns newNeuronId
+   * @throws {@link GovernanceError}
+   */
+  public splitNeuron = async ({
+    neuronId,
+    amount,
+  }: {
+    neuronId: NeuronId;
+    amount: ICP;
+  }): Promise<NeuronId> => {
+    const request = toSplitRawRequest({
+      neuronId,
+      amount: amount.toE8s(),
+    });
+
+    const { command } = await this.certifiedService.manage_neuron(request);
+    const response = command[0];
+
+    if (!response) {
+      throw new GovernanceError({
+        error_message: "Error updating neuron",
+        error_type: 0,
+      });
+    }
+
+    if ("Error" in response) {
+      throw new GovernanceError(response.Error);
+    }
+
+    if ("Split" in response) {
+      const neuron = response.Split.created_neuron_id[0];
+      if (neuron === undefined) {
+        // Edge case
+        throw new GovernanceError({
+          error_message: "Unexpected error splitting neuron",
```

Comment:
"Unexpected error splitting neuron" is used twiced, maybe worth using two different message to distinguish the error in case of support?

Reply:
Sure, makes sense

Reply:
done